bodo/transforms/distributed_analysis.py

```diff
@@ -2635,6 +2639,23 @@ def _analyze_call(
             _meet_array_dists(self.typemap, out_arrname, in_arrname, array_dists)
             return
 
+        if fdef == ("get_data", "bodo.libs.struct_arr_ext"):
+            in_arrname = rhs.args[0].name
+            if lhs not in array_dists:
+                _set_var_dist(self.typemap, lhs, array_dists, Distribution.OneD)
+            if in_arrname not in array_dists:
+                _set_var_dist(self.typemap, in_arrname, array_dists, Distribution.OneD)
+
+            lhs_dists, rhs_dist = array_dists[lhs], array_dists[in_arrname]
+            new_dist = Distribution(
+                min(min(x.value for x in lhs_dists), rhs_dist.value)
+            )
+            new_dist = _min_dist_top(new_dist, Distribution.OneD)
```

Comment:
This line seems unnecessary since `new_dist` is equal or lower than `OneD` but not a big deal.